doc/getting_started/getting_started_go.rst

```diff
@@ -245,3 +245,197 @@ To send bare Lua code for execution, use ``Eval``:
+        -   Yes (structure and marshall functions must be predefined in Go code)
+
+    *   -   Access tuple fields by names
+        -   Only if marshalled to structure
+        -   No
+        -   Only if marshalled to structure
+
+    *   -   `SQL <https://www.tarantool.io/en/doc/latest/reference/reference_sql/>`_ support
+        -   No (`#62 <https://github.com/tarantool/go-tarantool/issues/62>`_)
+        -   No (`#18 <https://github.com/viciious/go-tarantool/issues/18>`_, closed)
+        -   No
+
+    *   -   `Interactive transactions <https://www.tarantool.io/en/doc/latest/book/box/stream/>`_
+        -   No (`#101 <https://github.com/tarantool/go-tarantool/issues/101>`_)
+        -   No
+        -   No
+
+    *   -   `Varbinary <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`_ support
+        -   Yes (with in-built language tools)
+        -   Yes (with in-built language tools)
+        -   No (decodes to string by default, see `#6 <https://github.com/FZambia/tarantool/issues/6>`_)
+
+    *   -   `UUID <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`_ support
+        -   Yes
+        -   No
+        -   No
+
+    *   -   Decimal support
+        -   No (`#96 <https://github.com/tarantool/go-tarantool/issues/96>`_)
+        -   No
+        -   No
+
+    *   -   `EXT_ERROR <https://www.tarantool.io/ru/doc/latest/dev_guide/internals/msgpack_extensions/#the-error-type>`_
+            support
+        -   No
+        -   No
+        -   No
+
+    *   -   `Datetime <https://github.com/tarantool/tarantool/discussions/6244>`_ support
+        -   No (`#118 <https://github.com/tarantool/go-tarantool/issues/118>`_)
+        -   No
+        -   No
+
+    *   -   `box.session.push() responses <https://www.tarantool.io/ru/doc/latest/reference/reference_lua/box_session/push/>`_
+        -   No (`#67 <https://github.com/tarantool/go-tarantool/issues/67>`_)
+        -   No (`#21 <https://github.com/viciious/go-tarantool/issues/21>`_)
+        -   Yes
+
+    *   -   `Session settings <https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_space/_session_settings/>`_
+        -   No
+        -   No
+        -   No
+
+    *   -   `Graceful shutdown <https://github.com/tarantool/tarantool/issues/5924>`_
+        -   No
+        -   No
+        -   No
+
+    *   -   `IPROTO_ID (feature discovering) <https://github.com/tarantool/tarantool/issues/6253>`_
+        -   No
+        -   No
+        -   No
+
+    *   -   `tarantool/crud <https://github.com/tarantool/crud>`_ support
+        -   No
+        -   No
+        -   No
+
+    *   -   Connection pool
+        -   Yes (round-robin failover; no balancing; master discovering planned in `#113 <https://github.com/tarantool/go-tarantool/issues/113>`_)
+        -   No
+        -   No
+
+    *   -   Support retrying
+        -   No
+        -   No
+        -   No
+
+    *   -   Implicit reconnecting strategy
+        -   Yes
+        -   Yes
+        -   Yes
```

Comment:
I'll note that here it is not clear what behaviour we'll get at reconnecting for requests in the fly (and for requests added during the reconnect). Maybe it depends on certain request state: added into outgoing buffer, goes into network, etc.

@DifferentialOrange I think it is okay to don't dig here a lot and leave it as is, but if you can give details, it worth to add them here.

Reply:
I've tried to reproduce some cases and filed some issues

Reply:
FZambia says this is to be expected on his side: https://github.com/FZambia/tarantool/issues/7